src/main/sensors/battery.c

```diff
@@ -50,6 +50,8 @@
 
 #define ADCVREF 3300   // in mV
 
+#define MAX_ESC_BATTERY_AGE 10
```

Comment:
Yeah, `numRetriesSinceLastSuccessfulFrame` is probably better. But I think I will change it to a timestamp for the last successful frame at a later stage.